parlai/agents/memnn_extended/memnn_extended.py

```diff
@@ -0,0 +1,492 @@
+# Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
maybe call it "feedback" or "fp" or something more specific than "extended" ?

Reply:
It supports fp/rbi/rbi+fp/im so I wanted something more general. I can rename it to 'fp' if you think it won't be confusing.